tools/build_defs/native_tools/native_tools_toolchain.bzl

```diff
@@ -0,0 +1,37 @@
+ToolInfo = provider(
+    doc = "Information about the native tool",
+    fields = {
+        "path": """Absolute path to the tool in case the tool is preinstalled on the machine.
+Relative path to the tool in case the tool is built as part of a build; the path should be relative
+to the bazel-genfiles, i.e. it should start with the name of the top directory of the built tree
+artifact. (Please see the example "//examples:built_cmake_toolchain")""",
+        "label": """If the tool is preinstalled, must be None.
```

Comment:
1. The value of this field is a Target - let's call it "tool" or "target" instead of "label". I was confused - I thought this filed contains an actual Label object which would be incorrect.
2. Let's say I have a tool binary checked in - do we support that? (I guess that would be label + empty path)

Reply:
1. Renamed label to target
2. I added an example in examples/with_prebuilt_ninja_artefact, and mentioned it in README.